cumulus_library/study_parser.py

```diff
@@ -210,6 +210,46 @@ def _execute_drop_queries(
             cursor.execute(drop_view_table)
             query_console_output(verbose, drop_view_table, progress, task)
 
+    def load_builder_from_file(self, filename) -> (BaseTableBuilder, object):
```

Comment:
no logic changes here, just moved it out so it can be used for building all studies/building one studies